### PR TITLE
Update convert

### PR DESCRIPTION
**Pull request by luciolebrillante**

### CODE REVIEW

File: convert [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # To convert a file from jpg to pdf
 convert original.jpg converted.pdf
 
+# To rotate an image to 180 degrees
+convert original.jpg -rotate 180 converted.jpg
```
Comment (EmilySeville7cfg) [inline convert:5]:
```suggestion
convert <path/to/original.jpg> -rotate 180 <path/to/converted.jpg>
```

Maybe it's better and more obvious where arbitrary values can be placed.

**Review by EmilySeville7cfg — APPROVED** [review]
There are some suggestions could be applied. But they are optional because right now there are no such requirements.

File: convert (hunk 1 above)
Comment (luciolebrillante) [reply convert]: Thank you for your suggestion, it is a good idea.

File: convert [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 # To resize an image and simultaneously change its file type:
-convert original.jpg -resize 100x converted.png
+convert <path/to/original.jpg> -resize 100x <path/to/converted.png>
 
 # To resize all of the images within a directory:
 for file in `ls original/image/path/`;
     do new_path=${file%.*};
-    new_file=`basename $new_path`;
-    convert $file -resize 150 converted/image/path/$new_file.png;
+    $converted_file=`basename $new_path`;
```
Comment (EmilySeville7cfg) [inline convert:22]: Wrong shell syntax used: variables don't begin with a dollar sign like in Perl. :)

File: convert [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
 # To resize an image to a fixed width and height:
-convert original.jpg -resize 100x100 converted.jpg
+convert <path/to/original.jpg> -resize 100x100 <path/to/converted.jpg>
 
 # To resize an image and simultaneously change its file type:
-convert original.jpg -resize 100x converted.png
+convert <path/to/original.jpg> -resize 100x <path/to/converted.png>
 
 # To resize all of the images within a directory:
 for file in `ls original/image/path/`;
```
Comment (EmilySeville7cfg) [inline convert:20]:
```suggestion
for file in <path/to/original_image/*>`; do
```

File: convert [hunk 4]
```diff
@@ -1,27 +1,30 @@
...
     do new_path=${file%.*};
```
Comment (EmilySeville7cfg) [inline convert:21]:
```suggestion
for file in <path/to/original_image/*>`; do
    new_path="${file%.*}"
```

File: convert (hunk 2 above)
Comment (EmilySeville7cfg) [reply convert:22]: - Changes are suggested.
- `$converted_file=`basename $new_path`;` should be changed to `converted_file="$(basename "$new_path")"`.

Comment (luciolebrillante) [reply convert]: Added, thank you!

**Comment by luciolebrillante** [comment]
The changes I did and recommended by @EmilySeville7cfg about the name file are the same used in some other cheatsheet files.

I will not revert them, feel free to do it if you think it is better without them.

**Comment by EmilySeville7cfg** [comment]
@luciolebrillante, we are discussing format of cheatsheets [here](https://github.com/cheat/cheatsheets/pull/183#issuecomment-1324338507). U can join this discussion too. I think your opinion as a community feedback is important too.

**Pull request closed without merging**